src/Loader.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import oval from './oval.svg';
+import './Loader.css';
+import classNames from 'classnames';
+
+class Loader extends Component {
```

Comment:
I think this is a good candidate for a stateless component: https://facebook.github.io/react/docs/reusable-components.html#stateless-functions

You can manage the hide show based on the props, or maybe just in the container view, I'll leave a note there.